src/camera_transfer_state.rs

```diff
@@ -231,12 +231,23 @@ pub fn enter_camera_transfer_loop(
                             process::exit(1);
                         })
                         .unwrap();
-                    pin.set_interrupt(Trigger::RisingEdge, None)
-                        .map_err(|e| {
-                            error!("Unable to set pi ping interrupt: {e}");
-                            process::exit(1);
-                        })
-                        .unwrap();
+
```

Comment:
Does this fix the issue, or is it just something you're trying out?  Maybe the real solution is to just make sure we're not panicking on any of the threads we're spawning?